includes/table/class-following.php

```diff
@@ -0,0 +1,236 @@
+<?php
+/**
+ * Followers Table-Class file.
+ *
+ * @package Activitypub
+ */
+
+namespace Activitypub\Table;
+
+use Activitypub\Collection\Actors;
+use Activitypub\Collection\Following as FollowingCollection;
```

Comment:
```suggestion
use Activitypub\Collection\Following as Following_Collection;
```
Let's stick with snake case here?

Reply:
Fixed with https://github.com/Automattic/wordpress-activitypub/pull/1866/commits/ce80f5f70e2d9eafb5ff021aeb454a48dc12d493